client/homescreen/stats-overview/style.scss

```diff
@@ -12,10 +12,6 @@ $promo-actions-border-bottom: $light-gray-tertiary;
 	}
 }
 
-.woocommerce-stats-overview__footer {
-	border-top: 1px solid $outer-border;
-}
-
 .woocommerce-stats-overview__more-btn {
```

Comment:
Minor, but while we're here, the more button is very slightly off-center.  Barely noticeable when focused, but I think it should be a quick fix.